Examples/Physics_applications/laser_acceleration/PICMI_inputs_3d.py

```diff
@@ -48,114 +49,114 @@
 plasma_ymax = 20e-06
 plasma_zmax = None
 uniform_distribution = picmi.UniformDistribution(
-    density = plasma_density,
```

Comment:
I prefer spaces around `=` since it makes the text less dense and easier to read.

Reply:
I think PEP8 requires for arguments no spaces and spaces only for assignments.

Reply:
PEP8 does say no space around `=` for arguments, but only shows this when the arguments are all on one line. This is fine. But for multiline function calls, I find it much more readable to have spaces around `=`. It is inconsistent to require spaces around all operators (to spread things out) except in this one case where it would in fact make the code easier to read.

Reply:
You are right, I think it's a pragmatic choice in black. They do spaces for assignments and assignments in definitions (default values) but not in call site arguments. I kinda like it, but as all style choices it might come down to taste and what one is used to :)